frontend/src/app/app.component.html

```diff
@@ -1,3 +1,3 @@
 <app-loading-spinner></app-loading-spinner>
-<app-header *ngIf="!isAuthPage && isRouteNotFound"></app-header>
+<app-header *ngIf="!isWithoutHeaderPage && isRouteFound "></app-header>

```

Comment:
should hide the header not in this way, because as bigger our system became as much options we will need to write here

Pleas think about another way of hiding heder